v_4_4_0/files/conf/hardening.conf

```diff
@@ -10,3 +10,7 @@ net.ipv6.conf.all.accept_redirects = 0
 net.ipv6.conf.default.accept_redirects = 0
 # Increased mmapfs because some applications, like ES, need higher limit to store data properly
 vm.max_map_count = 262144
+# Ingress controller performance improvements
+# See https://github.com/kubernetes/ingress-nginx/issues/1939
+net.core.somaxconn=32768
+net.ipv4.ip_local_port_range="1024 65535"
```

Comment:
@corest 
This causes a problem when I tried to run this with the latest coreOS version:
```
Jun 12 13:12:54 localhost systemd[1]: Starting Apply Kernel Variables...
Jun 12 13:12:54 localhost systemd-sysctl[1006]: Couldn't write '"1024 65535"' to 'net/ipv4/ip_local_port_range': Invalid argument
Jun 12 13:12:54 localhost systemd[1]: systemd-sysctl.service: Main process exited, code=exited, status=1/FAILURE
Jun 12 13:12:54 localhost systemd[1]: systemd-sysctl.service: Failed with result 'exit-code'.
Jun 12 13:12:54 localhost systemd[1]: Failed to start Apply Kernel Variables.
```
The formatting likely has to be different.

Reply:
Create an issue for that and I'll check next week

Reply:
I'll take care, I am working on release for 1.14.3. 
Maybe I can get it working :D